### DISCUSSION TITLE
前端编译报错讨论

### DISCUSSION THREAD

**Discussion by kl7sn** [clickvisual/clickvisual#859] [Q&A] [answered]
@qianque7 @xiedeyantu 

前端编译报错可以在这里讨论，我们这边编译正常情况下 npm install 之后 yarn run start 就可以了。  
你可以截图发到这里，方便后续形成文档。

**Comment by xiedeyantu** [comment]
报错如下：
env: node: No such file or directory
make: *** [build.ui] Error 127

希望有个编译相关的指导文档，包括在各个系统下是否有不同，如Centos、Ubuntu、MacOS等主流系统。

**Reply by qianque7** [reply, marked as answer]
试试先确认clickvisual/ui/package.json和clickvisual/ui/yarn.lock文件是否为最新版，然后删除ui下的node_modules文件，在ui路径下执行yarn，成功后再执行yarn run start
